.github/scripts/icomoon_build.py

```diff
@@ -20,31 +20,31 @@ def main():
     runner = None
     try:
         args = arg_getters.get_selenium_runner_args()
-        new_icons = get_icons_for_building(args.devicon_json_path, args.token)
+        new_icons = get_icons_for_building(args.icomoon_json_path, args.devicon_json_path, args.token)
         if len(new_icons) == 0:
             sys.exit("No files need to be uploaded. Ending script...")
 
         print(f"There are {len(new_icons)} icons to be build. Here are they:", *new_icons, sep = "\n")
 
-        print("Begin optimizing files")
-        optimize_svgs(new_icons, args.icons_folder_path)
-
-        print("Updating the icomoon json")
-        update_icomoon_json(new_icons, args.icomoon_json_path)
-
-        icon_svgs = filehandler.get_svgs_paths(
-            new_icons, args.icons_folder_path, icon_versions_only=True)
-        runner = SeleniumRunner(args.download_path,
-                                args.geckodriver_path, args.headless)
-        runner.upload_icomoon(args.icomoon_json_path)
-        runner.upload_svgs(icon_svgs)
-
-        zip_name = "devicon-v1.0.zip"
-        zip_path = Path(args.download_path, zip_name)
-        runner.download_icomoon_fonts(zip_path)
-        filehandler.extract_files(str(zip_path), args.download_path)
-        filehandler.rename_extracted_files(args.download_path)
-        print("Task completed.")
+        # print("Begin optimizing files")
+        # optimize_svgs(new_icons, args.icons_folder_path)
+
+        # print("Updating the icomoon json")
+        # update_icomoon_json(new_icons, args.icomoon_json_path)
+
+        # icon_svgs = filehandler.get_svgs_paths(
+        #     new_icons, args.icons_folder_path, icon_versions_only=True)
+        # runner = SeleniumRunner(args.download_path,
+        #                         args.geckodriver_path, args.headless)
+        # runner.upload_icomoon(args.icomoon_json_path)
+        # runner.upload_svgs(icon_svgs)
+
+        # zip_name = "devicon-v1.0.zip"
+        # zip_path = Path(args.download_path, zip_name)
+        # runner.download_icomoon_fonts(zip_path)
+        # filehandler.extract_files(str(zip_path), args.download_path)
+        # filehandler.rename_extracted_files(args.download_path)
+        # print("Task completed.")
```

Comment:
Why you decided to remove this part? Was it for debugging or do you have a deeper explaination? :)

Reply:
Oh damn, that's definitely for debugging. I thought I uncommented it already